### PR TITLE
automated plotting 

### LINKED ISSUE #49: add capability for automating plotting

### PR DESCRIPTION
**Pull request by jrknezha**
This PR adds the bash scripts that are needed for automated plotting, cleans up the plotting to only pull the data which is relevant to the NNJA bucket, and includes updates that were necessary to run the automation scripts in particular handling the MySQL connection limits.

These scripts have been tested and the 3 day script is currently in use to populate the NNJA webpage graphics daily.

### CODE REVIEW

File: src/automation/run_3day_inventory.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -l
+# This script is for running the automated inventory for 
+# the past 3 days to get updates for lagged data and plotting
+# the new data.
+
+SATINFO_DIR=/home/Jessica.Knezha/obs-inventory/build_gsinfo/satinfo
```
Comment (frolovsa) [inline src/automation/run_3day_inventory.sh]: what if we generalize it by using ${USER} or some other environmental variable so we dont have to have these hard-codded values in a github repo?

File: src/automation/run_3day_inventory.sh [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+#run inventory 
+python3 auto_inventory.py -cat atmosphere -ago 3 -n_jobs 50 -work_dir $WORK_DIR 
+
+#run all plots in parallel
+python3 ../plotting/plot_mysql_dir_sensor_sat.py --sidb $SATINFO_DIR -o $OUTPUT_LOC &
+python3 ../plotting/plot_mysql_sensor.py -o $OUTPUT_LOC 
+python3 ../plotting/plot_mysql_sensor_sat_amv.py --sidb $SATINFO_DIR -o $OUTPUT_LOC &
+python3 ../plotting/plot_mysql_sensor_sat_geo.py --sidb $SATINFO_DIR -o $OUTPUT_LOC 
+python3 ../plotting/plot_mysql_sensor_sat_gps.py --sidb $SATINFO_DIR -o $OUTPUT_LOC &
+python3 ../plotting/plot_mysql_typ.py -o $OUTPUT_LOC
```
Comment (frolovsa) [inline src/automation/run_3day_inventory.sh:23]: I am curious to the logic of some of the calls running in the background and some are not?

**Review by frolovsa — CHANGES_REQUESTED** [review]
Thank you @jrknezha. I made a couple of comments/suggestions. I dont think they are blocking per say. But i would like to get your feedback on them. Thank you.

File: src/automation/run_3day_inventory.sh (hunk 2 above)
Comment (jrknezha) [reply src/automation/run_3day_inventory.sh:23]: I found that it ran more efficiently when there was only two at a time, probably due to IO overload and memory overload from processing all the SQL info. It also made it easier to read the output for errors.

File: src/automation/run_3day_inventory.sh (hunk 1 above)
Comment (jrknezha) [reply src/automation/run_3day_inventory.sh]: Great idea. I have updated the scripts and tested them, it's fixed now.

**Review by frolovsa — APPROVED** [review]

**Pull request merged**